src/include/wiredtiger.in

```diff
@@ -259,6 +259,25 @@ struct __wt_cursor {
 	 */
 	int __F(get_value)(WT_CURSOR *cursor, ...);
 
+        /*!
+	 * Get the raw key and value for the current record.
+	 *
+	 * @snippet ex_all.c Get the raw key and value for the current record.
+	 *
+	 * @snippet ex_all.c Set the cursor's record number key
+	 *
+	 * @param cursor the cursor handle
+	 * @param key pointer to an item that will contains the current record's raw key
+	 * @param value pointer to an item that will contains the current record's raw value
+	 * The API does not validate the argument types passed in.
```

Comment:
I don't think this line applies - the input types are typed here, so the argument types are valid.

Reply:
I'll remove the line.

Reply:
Done!